### ISSUE TITLE
Unlogged recording on a remote server

### ISSUE THREAD

**Issue by p91paul** [unloggedio/unlogged-sdk#30]
### Describe the bug

Following an error in the logs and inspecting source code, I discovered an upload feature in NetworkClient.
Apparently you can configure a server url where logs of your sessions will be uploaded.
The bug is in the documentation for not explaining how to use that.

Here is the error I saw:

[unlogged] failed to upload file: /home/myuser/.unlogged/sessions/selogger-output-20240124-155327969/index-00072-1706108307400.zip (No such file or directory)

### Reproduction steps

n/a

### Expected behavior

Documentation for the feature is present

**Comment by artpar** [comment]
Hi @p91paul 

We built a POC for the non-local pipeline of usage for unlogged-sdk (thing staging/pre-prod servers). The pipeline included

1. unlogged-sdk streaming log archives to a self-hosted unlogged-server 
2. processing of logs on the server + storage in s3/aerospike
3. plugin as client shows recorded tests and mocks from server to developer

While we did do a successful POC around the same earlier, we have realized there is a huge amount of effort we need to put in perfecting the local usage flow first. We will come back and finish the server pipeline in the coming months including the documentation.
